packages/test/test-end-to-end-tests/src/test/gc/gcUnreferencedTimestamp.spec.ts

```diff
@@ -21,19 +21,14 @@ import {
 	ITestDataObject,
 	TestDataObjectType,
 } from "@fluid-private/test-version-utils";
-// eslint-disable-next-line import/no-internal-modules
-import { detectOutboundRoutesViaDDSKey } from "@fluidframework/container-runtime/test/gc";
 import { defaultGCConfig } from "./gcTestConfigs.js";
 import { getGCStateFromSummary } from "./gcTestSummaryUtils.js";
 
 /**
  * Validates that the unreferenced timestamp is correctly set in the GC summary tree. Also, the timestamp is removed
  * when an unreferenced node becomes referenced again.
- *
- * Run in FullCompat to get coverage of DataStoreRuntime/ContainerRuntime n/n-1 compatibility
- * (skip all other compat types)
  */
-describeCompat("GC unreferenced timestamp", "FullCompat", (getTestObjectProvider, apis) => {
+describeCompat("GC unreferenced timestamp", "2.0.0-rc.1.0.0", (getTestObjectProvider, apis) => {
```

Comment:
For compatibility sake, I'm assuming the attach ops should still be able to be processed by older versions. Especially now that we may get some much bigger compatibility swings.

Reply:
Yes the new attach ops should be. I didn't think about test coverage for this, it seems vacuously true (just adding a blob). I'll consider it...

Reply:
I did confirm once that running the bulk of these tests under FullCompat works fine.  I'm leaving this as NoCompat for now, see other thread in this file about compat